compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AMDAIERT.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#pragma once
```

Comment:
Could you use the `#ifndef IREE_AMD_AMDAIE_RT_H_` header guards instead in consistency with upstream?